ilp-core/src/main/java/org/interledger/core/InterledgerPacketType.java

```diff
@@ -0,0 +1,52 @@
+package org.interledger.core;
+
+import static java.lang.String.format;
+
+import java.util.Objects;
+
+/**
+ * Per the IL-RFC-27, ILPv4 packets have three types: 12 for Prepare, 13 for Fulfill, and 14 for Reject. This enum
+ * provides some type safety for code that might need this information.
+ *
+ * @see "https://github.com/interledger/rfcs/tree/master/0027-interledger-protocol-4"
+ */
+public enum InterledgerPacketType {
```

Comment:
Created #245 to cover this class. If anyone is interested in picking it up, just assign that issue to yourself.

Reply:
This was resolved in 0144c63